sql/item_mods.sql

```diff
@@ -37387,8 +37386,6 @@ INSERT INTO `item_mods` VALUES (28665,29,6);
 INSERT INTO `item_mods` VALUES (28665,109,106);
 INSERT INTO `item_mods` VALUES (28665,161,-8);
 INSERT INTO `item_mods` VALUES (28670,1,1);
-INSERT INTO `item_mods` VALUES (28757,23,14);
```

Comment:
I think the original author meant 28575
http://www.ffxiah.com/item/28575/choj-band